lib/diagnostic.rb

```diff
@@ -91,13 +91,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr = [12, 34, 56, 67].slice(-2, 2)'
```

Comment:
`arr.pop(2)`  There are other ways to do this, but your solution returns those two elements, we wanted to remove them to see something like `'[12, 34, nil, nil, 99]'`